From de8eb47bbd95d48126d7c1e6d949e410963fbf9f Mon Sep 17 00:00:00 2001
From: Holger Freyther <ich@holger-freythers-macbook-pro.local>
Date: Mon, 10 Nov 2008 23:13:01 +0100
Subject: [PATCH] [mediawiki] Disable that for better debugging/development

    Okay, I can not read. I was not supposed to touch the DefaultSettings..

    disable caching. this makes things dog slow but allow to hack on that
    stuff.
---
 includes/DefaultSettings.php |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/includes/DefaultSettings.php b/includes/DefaultSettings.php
index 88aa08b..bc27ec0 100644
--- a/includes/DefaultSettings.php
+++ b/includes/DefaultSettings.php
@@ -1029,7 +1029,7 @@ $wgUseCategoryBrowser   = false;
  * This can provide a significant speedup for medium to large pages,
  * so you probably want to keep it on.
  */
-$wgEnableParserCache = true;
+$wgEnableParserCache = false;
 
 /**
  * Append a configured value to the parser cache and the sitenotice key so
-- 
1.6.0.2

